tests/spec/fs.lstat.spec.js

```diff
@@ -47,3 +47,29 @@ describe('fs.lstat', function() {
     });
   });
 });
+
+describe('fs.promises.lstat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fsPromises = util.fs().promises;
+    expect(typeof fsPromises.lstat).to.equal('function');
+  });
+
+  it('should return an error if path does not exist', function() {
+    var fsPromises = util.fs().promises;
+    var fsStats;
+    fsPromises.lstat('/tmp')
```

Comment:
Add a `return` statement to this line, so you return your Promise for this test.